source/WindowScroller/WindowScroller.css

```diff
@@ -0,0 +1,3 @@
+.disabled > * {
+  pointer-events: none !important;
```

Comment:
Do you actually mean `.disabled *`?

Hm... either way, `> *` is not a very performant selector so I'm pretty reluctant to use it, and definitely not with such a generic class name as `disabled`. RV classes are named so as to (hopefully) not conflict with any other global CSS names.

Also related- RV has also been shifting away from any CSS-based styles (with the exception of `Table`). Basically anything that's functional is defined via an inline style; anything that's presentational (only `Table`) is defined via CSS to make it easier to override in a variety of ways (without requiring use of `!important`).